### PR TITLE
Add Chrome Scopes

### PR DESCRIPTION
**Pull request by jasonLaster**
### Summary of Changes

* publishing this as it has sat on my computer for a couple weeks
* it adds an optional scopeChain field to frame
* it adds a new chrome scopes component
* I'll work on converting firefox scopes to the scope chain notion next

### Test Plan

- [x] doesn't break firefox
- [x] works for chrome

### Screenshots/Videos (OPTIONAL)

screen shot 2016-12-13 at 10 36 23 am

### CODE REVIEW

File: packages/devtools-client-adapters/src/chrome/events.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 const { Source, Location, Frame } = require("../tcomb-types");
 
-let actions;
-let pageAgent;
+let actions = undefined;
```
Comment (jasonLaster) [inline packages/devtools-client-adapters/src/chrome/events.js]: `let x` doesn't work :)

Comment (clarkbw) [inline packages/devtools-client-adapters/src/chrome/events.js]: Why?  The default value should be `undefined`, right?

File: src/components/utils/Svg.css [hunk 2]
```diff
@@ -50,3 +50,29 @@ html[dir="rtl"] .worker svg,
 html[dir="rtl"] .refresh svg {
     margin-left: 5px;
 }
+
```
Comment (clarkbw) [inline src/components/utils/Svg.css]: 👍 for this move

**Comment by clarkbw** [comment]
> it adds a new chrome scopes component

Why choose this path?  I'm guessing our scopes are fairly tied to the actors and even tcomb.

Just wondering since this is the first split like this.

**Comment by jasonLaster** [comment]
This is blocked by the kill-packages PR.

**Comment by jasonLaster** [comment]
Here's a gif of what it looks like for the record:

File: src/reducers/pause.js [hunk 3]
```diff
@@ -169,10 +169,17 @@ function getSelectedFrame(state: OuterState) {
   return frames && frames.find(frame => frame.id == selectedFrameId);
 }
 
+// NOTE: currently only used for chrome
+function getScopes(state: OuterState) {
```
Comment (clarkbw) [inline src/reducers/pause.js]: I think this should be `getChromeScopes` so it is more obvious that is is only a single target method.

**Pull request merged**